chia/rpc/wallet_rpc_client.py

```diff
@@ -242,12 +243,12 @@ async def get_coins(
 
     async def spend_clawback_coins(
         self,
-        merkle_coin_ids: List[str],
+        coin_ids: List[str],
```

Comment:
I would have expected this to be `List[bytes32]` and that we would turn them into hex when creating the request message below

Reply:
Fixed